test/integration/framework/master_utils.go

```diff
@@ -129,21 +129,21 @@ func startMasterOrDie(masterConfig *master.Config, incomingServer *httptest.Serv
 
 	if masterConfig == nil {
 		masterConfig = NewMasterConfig()
-		masterConfig.GenericConfig.OpenAPIConfig = genericapiserver.DefaultOpenAPIConfig(openapi.GetOpenAPIDefinitions, openapinamer.NewDefinitionNamer(legacyscheme.Scheme))
-		masterConfig.GenericConfig.OpenAPIConfig.Info = &spec.Info{
-			InfoProps: spec.InfoProps{
-				Title:   "Kubernetes",
-				Version: "unversioned",
-			},
-		}
-		masterConfig.GenericConfig.OpenAPIConfig.DefaultResponse = &spec.Response{
-			ResponseProps: spec.ResponseProps{
-				Description: "Default Response.",
-			},
-		}
-		masterConfig.GenericConfig.OpenAPIConfig.GetDefinitions = openapi.GetOpenAPIDefinitions
-		masterConfig.GenericConfig.SwaggerConfig = genericapiserver.DefaultSwaggerConfig()
 	}
+	masterConfig.GenericConfig.OpenAPIConfig = genericapiserver.DefaultOpenAPIConfig(openapi.GetOpenAPIDefinitions, openapinamer.NewDefinitionNamer(legacyscheme.Scheme))
```

Comment:
@jennybuckley if you can please give this a look.
This was the way i found to solve missing OpenAPIModels on our integration tests (apply_test.go).
Quite sure its not the right change and breaking a lot of different stuff. Just didn't find a better way in the short time. -> need help :-)